spec/controllers/hq/admin_users_controller_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe Hq::AdminUsersController, type: :controller do
   let(:admin_users) { build_stubbed_list :admin_user, 2 }
+  let(:admin_user) { admin_users.first }
```

Comment:
Suggestion: By `using build_stubbed :admin_user` instead of `admin_users.first` we are not creating the second unnecessary `admin_user` from `build_stubbed_list`. Also decoupling things as much as possible is a good practice.